dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeInitializer.java

```diff
@@ -99,4 +113,27 @@ private void saveFavoritePageFields(final ContentTypeAPI contentTypeAPI, final C
             Logger.warnAndDebug(this.getClass(), e);
         }
     }
+
+    private void checkDefaultPermissions(final ContentType savedContentType) {
+
+        if (DO_DEFAULT_PAGE_PERMISSIONS) {
+
+            try {
+
+                final int permissionType = PermissionAPI.PERMISSION_USE | PermissionAPI.PERMISSION_EDIT |
+                        PermissionAPI.PERMISSION_PUBLISH | PermissionAPI.PERMISSION_EDIT_PERMISSIONS;
+                final Role backendRole = APILocator.getRoleAPI().loadCMSOwnerRole();
+                if (!APILocator.getPermissionAPI().doesRoleHavePermission(
+                        savedContentType, permissionType, backendRole)) {
+
+                    Logger.info(this, "Adding default permissions to the Favorite Page Content Type...");
```

Comment:
Let's change this to `DEBUG`

Reply:
why debug, i want to see it on the begging, it does not happen all the time